WordPress/WordPressTest/ReaderPostBuilder.swift

```diff
@@ -0,0 +1,17 @@
+@testable import WordPress
+
+/// Builds a `ReaderPost`
+///
+class ReaderPostBuilder {
```

Comment:
Notice this is no longer a `PostBuilder` subclass. It was not using any of the `PostBuilder` methods and was overriding the `init` without calling `super`. That is, it was already a stand alone class for all effect and purpose.